src/mapml.css

```diff
@@ -776,3 +776,45 @@ label.mapml-layer-item-toggle {
   ) {
   outline: 0!important;
 }
+
+
+/**
+* Feature Index
+ */
+.mapml-feature-index-box {
```

Comment:
We should add the `.mapml-feature-index-box` selector to this declaration:

https://github.com/Maps4HTML/Web-Map-Custom-Element/blob/9f1d807e5cac3201512d94ac8c8bddee1c52afc4/src/mapml.css#L432-L439

because currently it's possible to click the feature index box with a mouse pointer (thus preventing underlying features from receiving the click).